irohad/network/network_api.h

```diff
@@ -73,15 +53,16 @@ namespace iroha {
        * Method spreads transaction to other members of  a network
        * @param tx - transaction for propagation
        */
-      virtual void propagate_transaction(iroha::dao::Transaction &tx) = 0;
+      virtual void propagate_transaction(dao::Transaction &tx) = 0;
```

Comment:
No, this interface about transferring transaction to network. Under the hood it will be call consensus.share(tx) (or similar method) and share method sharing it to ordering service